MANUAL.txt

```diff
@@ -578,6 +578,16 @@ Reader options
     printed in some output formats) and metadata values will be escaped
     when inserted into the template.
 
+`--metadata-file=`*FILE*
+
+:   Read metadata from the supplied YAML (or JSON) file.
+    This option can be used with every input format, but string
+    scalars in the YAML file will always be parsed as Markdown.
```

Comment:
Also in a JSON file?

Reply:
Yes, since YAML is a superset of JSON and we don't check the file extension, we get that for free!